frontend/src/index.css

```diff
@@ -7,6 +7,15 @@ body {
     -moz-osx-font-smoothing: grayscale;
 }
 
+button {
```

Comment:
Why is this rule in the CSS file for the index?
Since we try to keep the the styling either in or next to the component.  Maybe extract this rule to a singular file `MapMarker.css` and place it right next to the tsx file.

Reply:
Sure will do that

Reply:
removed the styling and used the MUI component.